bootstrap/bootstrap.sh

```diff
@@ -28,8 +29,54 @@ sudo apt-get update
 sudo apt-get -y dist-upgrade
 
 #----nodejs install
-curl -sL https://deb.nodesource.com/setup_7.x | sudo -E bash -
+curl -sL https://deb.nodesource.com/setup_6.x | sudo -E bash -
 sudo apt-get install -y nodejs
 
-#----alsa install
-sudo apt-get install -y alsa-base alsa-utils
+#----install official requirements
+sudo apt-get install -y alsa-base alsa-utils libasound2-dev
+
+#----install missing pigpio in Raspbian Lite (the command npm install pigpio will be exec by package.json)
+sudo apt-get install pigpio
+
+#----install git and download tjbot
+if [ ! -d "${TJBOT_FOLDER}/recipes/conversation" ]; then
+    sudo apt-get install -y git
+
+    rm -Rf /home/pi/git/tjbot
+    git clone https://github.com/ibmtjbot/tjbot.git /home/pi/git/tjbot
+
+    TJBOT_FOLDER='/home/pi/git/tjbot'
+fi
+
+#----install conversation (to install it will resolve tjbotlib and other dependencies)
+cd $TJBOT_FOLDER
+cd recipes/conversation
+echo "path: $PWD"
+
+sudo npm install --unsafe-perm -g node-gyp
```

Comment:
@fivdi it is because I'm in an IBM traveling and don't have rpi here. So I get log from my last try that is related with tests folder. But at conversation the behavior is the same: permission errors (even using sudo) that could be fixed using `--unsafe-perm` and also a lot of warnings and deprecated messages. I used "sudo npm install" in that last try. That is true.

Yeah, I know at the end test.servo.js is successfully called. But that a lot of warning / deprecated messages (https://gist.github.com/fivdi/f7a45e595d3039cb0c5d631425bb080d) give the impression that are something wrong. A end user could thing that it is broken.

So `sudo npm install --unsafe-perm -g readline-sync` run tests fine without errors and also `sudo npm install --unsafe-perm -g node-gyp ` do the same with conversation.

I tried to know what is happening with `sudo npm install` or `npm install` but I didn't know. If we have a clean messages using `npm install` sure it is better.